### ISSUE TITLE
The Quill does not save/load from Oricutron

### ISSUE THREAD

**Issue by ChemaEnguita** [pete-gordon/oricutron#119]
The Quill (an adventure writer application) uses its own custom load/save routines, which seem to fail in Oricutron.

I have tried several options:
* Simply select save and a filename (with .tap extension and without it), as requested by the program. No TAP file is generated.
* Check if it was adding it to the quill.tap. I am not completely sure, but I think it does not try to add it even if the tap name is displayed on the status as if writing there. In any case, I cannot load it back. As the program can search for data with a given name, I selected another tap then the quill.tap file again (is there an easy way to rewind?) and tried to LOAD it. It FOUND the program data chunks and nothing else. Besides the file was apparently not modified.
* I found no way to create an empty file from the emulator's menu whatsoever.
* Create a dummy tape with an empty program and CSAVE"", then selecting it in Oricutron before saving the game data. It did not work, even if the loading routine can search for a chunk with a given name, ignoring the rest.
* Use the F9 option to start recording on a tape (ort), then save, then press F9 again. Some kind of dummy data appears on the file, but it seems to me that it is created when the ort image is created, but no actual data is stored. When trying to restore the data, I tried to select the ort file and all I got is a "Searching.." message.

So, basically I can't find a way to make Oricutron save the work on a new file so it can be loaded back. It may be that Quill is using some kind of ROM calls which are not trapped by the emulator, or use its own routines or that I am being dumb, but there is no obvious way to do something as simple as that.

**Comment by assinie** [comment]
I found a way.
After loading quill.tap:
- disable the option "Turbo Tape" in the hardware menu
- select "Save tape output"
- go back to the quill and save your datas (you can save many times)
- go to Oricutron menu and select "Stop tape recording"
- select  "Insert tape" and select the ort file from step 2
- go back to the quill
- you can load your datas
